scripts/zones/QuBia_Arena/mobs/Hell_Fly.lua

```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+-- Area: Qu'Bia Arena
+--  Mob: Hell fly
+-- KSCNM: Infernal Swarm
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.CHARMABLE, 1)
+    mob:setMod(xi.mod.SLEEPRES, 250)
```

Comment:
proof is linked in the commit. Should I also link in the PR comment?

Reply:
It was beelzebub that has the sleep resistance. I pushed a commit correcting this.

Reply:
you got it. I'll be more rigorous in linking the basis on which I make the changes.

Reply:
FYI, sleepres doesn't go that high. stops at 100 last time I looked at how it was being used.

Reply:
After testing, 100 seems to be 10%. I can only receive immunity from the mob at 1000. Aerec and I spoke of this issue in voice earlier.

Reply:
@Xaver-DaRed did you rework stat res to 1000 scale in you magic rework and I missed it?

Reply:
I don't remember touching them. My rework hasnt reached status effects yet

Reply:
I'm not seeing anything in our existing globals implying 1000 base, in fact I see code implying the 100 I expected.

> A quick test using 100 just now and I can still land everything

I'm not surprised, status res has never been retail correct and it may be that throwing massive numbers at it bumps up meva enough to be finally noticeable. It's not supposed to be just meva tho. 

https://www.bg-wiki.com/ffxi/Resist_Sleep
> - Causes the Sleep status effect to be fully resisted a certain percentage of the time.
> - At the moment, it is assumed that Resist Sleep +1 grants a 1% increase in this change to resist with an unknown cap. It is suspected the cap is lower than 100%.

https://ffxiclopedia.fandom.com/wiki/Resist_Sleep
> - Each upgrade of this trait further increases character's resistance.
> - This trait greatly shortens the duration of [sleep](https://ffxiclopedia.fandom.com/wiki/Sleep) type status effects by increasing the chance of half duration resists.
> - Gives a small chance to completely resist the effect of [sleep](https://ffxiclopedia.fandom.com/wiki/Sleep) status. When it procs the text will be preceeded by "Resist!" in a similar fashion to [Cover](https://ffxiclopedia.fandom.com/wiki/Cover).

Reply:
double checked how we are using them in items and existing mobs, it's all expecting 100 scaled in our trunk. if it somehow is not, and as I have seen no use case for fractions of a %, any place making it 1000 scale should probably be set to 100 rather than updating multiple dozens of mobs and items.

Reply:
@dallano for what it's worth I've complained the resist system was pretty much broken since about 2014 under old dsp. It's on our radar to redo in LSB but other things have taken priority. What I think is happening is your sleepres is just being added as flat meva.